### PR TITLE
BZ 1946313: Review&Create failing valition hint

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: src/components/clusterConfiguration/ReviewValidations.tsx [hunk 1]
```diff
@@ -67,7 +67,10 @@ const FailingValidation: React.FC<{
 
   let fix;
   const step = findValidationFixStep({ id: validation.id, clusterGroup, hostGroup });
-  if (step) {
+  if (step === 'review') {
```
Comment (jtomasek) [inline src/components/clusterConfiguration/ReviewValidations.tsx:70]: Maybe we should just remove the 'all-hosts-are-ready-to-install' from `reviewStepValidationsMap`? User is not able to fix anything in the review step, so it makes no sense to include that validation there.

Doing so would highlight that validation as failed and link to a step would not show up as expected.

File: src/components/clusterConfiguration/ReviewValidations.tsx [hunk 2]
```diff
@@ -67,7 +67,10 @@ const FailingValidation: React.FC<{
...
+    // no sooner step, so the user can not do anything about it ...
+    fix = 'Please wait till all validations are finished.';
```
Comment (jtomasek) [inline src/components/clusterConfiguration/ReviewValidations.tsx:72]: This is not exactly true as `all-hosts-are-ready-to-install` validation is failed and won't get updated.

**Review by jtomasek — CHANGES_REQUESTED** [review]

File: src/components/clusterConfiguration/ReviewValidations.tsx (hunk 2 above)
Comment (mareklibra) [reply src/components/clusterConfiguration/ReviewValidations.tsx:72]: Asking backend to confirm, but so far that was my understanding, so the fix would be ok. Let's see their response.

File: src/components/clusterConfiguration/ReviewValidations.tsx (hunk 1 above)
Comment (mareklibra) [reply src/components/clusterConfiguration/ReviewValidations.tsx:70]: But we need to disable `Install` in that case, don't we?

File: src/components/clusterConfiguration/ReviewValidations.tsx (hunk 2 above)
Comment (mareklibra) [reply src/components/clusterConfiguration/ReviewValidations.tsx:72]: confirmed, just temporary flow till the validation is re-executed

**Review by jtomasek — APPROVED** [review]

**Pull request merged**